docs/contribution/community-components.md

```diff
@@ -0,0 +1,299 @@
+  easier and thus reduce design and development effort. We also want to
+  encourage community members to contribute back, not just to Carbon, but to the
+  community components they're using as well. Just because core Carbon won't be
+  maintaining these, doesn't mean you can't contribute to them!
+
+- **Quality control:** We want to make contributing easy, but want to encourage
+  improvement, maintenance and contribution. We've created a recommended
+  checklist of comprehensive design and development guidelines. While there is
+  no requirement to meet any or all of these guidelines, component quality will
+  be displayed based off of how many guidelines you meet. Evaluation of level of
+  completeness is based on your assessment.
+
+  _If we're not required to meet any of the guidelines, why would we want to
+  create more work for ourselves by trying to meet most of them?_
+
+  Good question. Carbon will be endorsing components that meet 80% or more of
+  the guidelines as being on the same design and development level as core
+  Carbon components. This means that we will prioritize them when we surface
+  these components on our website. It is our goal that by endorsing these and
+  surfacing them more notably, more users will use and contribute to your
+  component and make it easier for you to maintain it.
+
+- **Maintainers:** The core Carbon team will not be responsible for maintaining
+  community components. The responsibility to maintain them will fall on the
+  team who contributed the component.
+
+<img src='./images/contribution_repo_diagram.png' alt='community components model diagram'/>
+
+The diagram above illustrates one way we could implement this vision for
+community contribution. In this model we would need both a Core Carbon and a
+Community monorepo. The Community repo would consume packages from Core Carbon
+repo and both monorepos would need to run on the same spec model (this has not
+been implemented yet). In order to accurately represent downloads and activity
+for a specific component on the Carbon website, components would need to live in
+the Community monorepo as packages.
+
+### Component checklist
+
+This check-list consists of a series of agreed upon acceptance criteria (put
+forth by Carbon designers and developers) that lives in the `package.json` file
+as a set of boolean config objects. Each spec will need some meta data (i.e.
+first major release when it was introduced) before we can badge it.
+Contributions can then be evaluated according to the completion of the spec.
+Some of the criteria can be tested through automation, some will be
+self-reported.
+
+In order to contribute a community component, the only _truly_ required items
+are the basic guidelines. This allows us to give you a unique package directory
+in the monorepo. Reaching 100% completeness means meeting all of the design and
+development guidelines on this list.
+
+**Basic (always required)**
```

Comment:
For communities that have more rigorus requirements would be we be able to extend the this list for our specific package? (regulated environments for example have more testing/documentation that is required).